### ISSUE TITLE
redirect Terms & Privacy URL currently used in iOS app 3.0.2

### ISSUE THREAD

**Issue by holgerd** [sozialhelden/wheelmap#395]
In the iOS app Version 3.0.2 (build 201) two "old" links still exist:

1. http://blog.wheelmap.org/was-ist-wheelmap/terms/
- This currently redirects to https://community.wheelmap.org/impressum/nutzungsbedingungen/ 
- But it should redirect to https://news.wheelmap.org/nutzungsbedingungen/

2. blog.wheelmap.org/was-ist-wheelmap/privacy/
- This currently redirects to https://community.wheelmap.org/en/imprint/privacy/
- But it should redirect to https://news.wheelmap.org/datenschutz/

**Comment by holgerd** [comment]
cc @Svenyo

**Comment by Xylakant** [comment]
shouldn't those links be fixed in the iphone app?

**Comment by holgerd** [comment]
yes they will. But we need redirects as well for those using the current app version.

**Comment by Xylakant** [comment]
ok, so the best would be to redirect these urls at the cloudflare level, but we've run out of page rules. so either we save some page rules (see for example #386) or we buy some more.

The alternative would be to redirect at the webserver level, which is less efficient. This would require four changes:

* [ ] redirect http://blog.wheelmap.org/was-ist-wheelmap/terms/ to  https://news.wheelmap.org/nutzungsbedingungen/ in the old prod/new prod environment
* [ ] redirect http://blog.wheelmap.org/was-ist-wheelmap/privacy/ to https://news.wheelmap.org/datenschutz/ in the old prod/new prod environment
* [ ] redirect https://community.wheelmap.org/impressum/nutzungsbedingungen/ to https://news.wheelmap.org/nutzungsbedingungen/ in the wordpress blog.
* [ ] redirect https://community.wheelmap.org/en/imprint/privacy/ to https://news.wheelmap.org/datenschutz/ in the wordpress blog.

**Comment by holgerd** [comment]
Yes we need this.

@100miles can you do this?

**Comment by 100Miles** [comment]
yep

**Comment by Hoverbear** [comment]
Oh hi, @100miles =D

**Comment by Xylakant** [comment]
@holgerd do what? There are two alternatives on the table. The second alternative actually requires the wordpress team to do some of the changes.

**Comment by holgerd** [comment]
@Xylakant imho we should  redirect at the webserver level. Since the old iPhone app is already "out there in the wild" we cannot change the URL there.

**Comment by Hoverbear** [comment]
Currently two of the redirects listed already happen. 

The redirect " redirect https://community.wheelmap.org/en/imprint/privacy/ to https://news.wheelmap.org/datenschutz/ in the wordpress blog." seems to me to be not right. Currently it directs you to https://news.wheelmap.org/en/privacy/ which seems correct. I'm marking that also done.

http://blog.wheelmap.org/was-ist-wheelmap/privacy/ currently redirects to https://news.wheelmap.org/en/privacy/ which is incorrect, should be https://news.wheelmap.org/datenschutz/ .
